src/DoubleTheDonation/FormExtension/Actions/ConvertBlockToField.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace GiveDoubleTheDonation\DoubleTheDonation\FormExtension\Actions;
+
+use Give\Donations\Models\Donation;
+use Give\Framework\Blocks\BlockModel;
+use Give\Framework\FieldsAPI\Contracts\Node;
+use Give\Framework\FieldsAPI\Exceptions\EmptyNameException;
+use GiveDoubleTheDonation\DoubleTheDonation\FormExtension\Field as DoubleTheDonationField;
+use GiveDoubleTheDonation\DoubleTheDonation\Helpers\DoubleTheDonationApi;
+
+/**
+ * @unreleased
+ */
+class ConvertBlockToField
```

Comment:
Can we please add a unit test for this 😄

Reply:
Definitely not in this PR :smiley: